main.go

```diff
@@ -24,27 +37,23 @@ func init() {
 		Credentials: credentials.NewStaticCredentials("AKID", "SECRET", ""),
 	})
 	if err != nil {
-		fmt.Println("Couldn't get AWS access")
-		panic("Internal Error")
+		log.Fatalf("couldn't get AWS access: %v", err.Error())
```

Comment:
Don't we need to return here?

Reply:
you are causing app to exit with `Fatal`

Reply:
shouldn't it though?

Reply:
doesn't need to return.

```
// Fatalf is equivalent to Printf() followed by a call to os.Exit(1).
func Fatalf(format string, v ...interface{}) {
	std.Output(2, fmt.Sprintf(format, v...))
	os.Exit(1)
}
```

Reply:
oh that is weird.  Never seen a log function do anything except for print a log.  Is this common in GO?